src/user-management/user-management.ts

```diff
@@ -137,7 +145,14 @@ const toQueryString = (options: Record<string, string | undefined>): string => {
 };
 
 export class UserManagement {
-  constructor(private readonly workos: WorkOS) {}
+  private jwks;
+
+  constructor(private readonly workos: WorkOS) {
+    // Set the JWKS URL. This is used to verify if the JWT is still valid
+    this.jwks = createRemoteJWKSet(
+      new URL(this.getJwksUrl(process.env.WORKOS_CLIENT_ID as string)),
+    );
```

Comment:
The client ID handling is a little awkward. We pass it in everywhere it's needed, but here we can either:
* Pass the environment variable from `workos.ts`
* Pull the environment variable here

Reply:
Problem here is that we're assuming that `process.env.WORKOS_CLIENT_ID` is set, which we can't guarantee. I think the right way to solve this is to have `clientId` as part of the WorkOS constructor that we can then pull from, something like:

```ts
this.clientId = this.options.clientId || process?.env.WORKOS_CLIENT_ID;
```

We could then throw an error if there's no client ID, similar to how we do it with the API key. You'd set the client ID on init:

```ts
const workos = new WorkOS('sk_test_123', {
  clientID: 'client_123',
});
```

which we could then export to the rest of the codebase via something like `this.workos.clientId`. Bonus is that you'd never have to manually pass in a client ID.